PatriotWolf/react-nest
# Feature/server data seed

# What does this PR do?

Base for data seeding simple fetch

## Type of change


## Changes

- Base Setup for seeding
- Base Setup for `GET`

## Feature Flags

## How to test (for reviewers)
1. At root of project, run `yarn install`.
2. Run `yarn seed on root`
3. Run `yarn dev`
4. Using localhost:3004 on browser, use `api/brands`, `api/colors`, `api/products` as `GET` request.
5. Listing should be display on 

## Demo

![image](https://github.com/PatriotWolf/react-nest/assets/23714533/23f636a9-01d2-4ad4-a23d-a472f9fbb848)
![image](https://github.com/PatriotWolf/react-nest/assets/23714533/7bdef065-1945-46d4-bff5-403585d3184a)
